### PR TITLE
Uplink Adjustment

### PR DESCRIPTION
**Pull request by Geevies**
* Lowered the cost of various uplink items across the board.
* The sideeffect b-gone injector box on the uplink now spawns with four injectors.
* Removed the AMR and its ammo from the uplink.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Review by Chada-1 — APPROVED** [review]

**Comment by GMR25** [comment]
If the Space Suit is being lowered to 2 TCs, should I have another look at the cost of the (traitor) gimmick voidsuits again? A major factor of my repricing of them was due to the Red Space Suit generally being equal or better for much cheaper.

File: code/datums/uplink/devices and tools.dm [hunk 1]
```diff
@@ -125,7 +125,7 @@
 
 /datum/uplink_item/item/tools/syndieborg
 	name = "Syndicate Cyborg Teleporter"
-	item_cost = 35
+	item_cost = 30
```
Comment (NonQueueingMatt) [inline code/datums/uplink/devices and tools.dm:128]: I don't know how I feel about allowing traitors to have a syndieborg. They're really fucking powerful.

File: code/datums/uplink/exosuit.dm [hunk 2]
```diff
@@ -10,7 +10,7 @@
 /datum/uplink_item/item/exosuit/heavy
 	name = "Heavy Exosuit Dropper"
 	desc = "A device that can be used to drop in a heavy exosuit. Can only be used outside station areas, unless emagged, which is hazardous."
-	item_cost = 35
+	item_cost = 30
```
Comment (NonQueueingMatt) [inline code/datums/uplink/exosuit.dm:13]: Same deal.

File: code/datums/uplink/grenades.dm [hunk 3]
```diff
@@ -12,29 +12,29 @@
...
 	name = "5xEMP Grenades"
-	item_cost = 6
+	item_cost = 5
 	path = /obj/item/storage/box/emps
 	desc = "A box of five grenades that cause a risky EMP explosion, capable of toggling headsets off, permanently destroying IPC units and draining a stationbound completely."
 	
 /datum/uplink_item/item/grenades/frag
 	name = "5xFrag Grenades"
-	item_cost = 12
+	item_cost = 10
```
Comment (NonQueueingMatt) [inline code/datums/uplink/grenades.dm:33]: Absolutely not.  Frag grenades are busted as-is.

**Review by Doxxmedearly — APPROVED** [review]
Hesitant to do both TC increase and cost decrease on so many items. I support things that are rarely used or helpful to stay ahead of the game (Radio keys, chameleon kits, supply beacon drops, etc) but am unsure about so many combat-related things having a reduction. Technically fine as-is but consider reducing the amount of TC discounts for combat items.

**Pull request merged**